src/__tests__/Navbar.test.js

```diff
@@ -0,0 +1,34 @@
+import { render } from '@testing-library/react';
+import { Provider } from 'react-redux';
+import { BrowserRouter as Router } from 'react-router-dom';
+import configureStore from 'redux-mock-store';
+import Navbar from '../components/Navbar';
+
+const mockStore = configureStore([]);
+
+describe('Navbar', () => {
+  test('Navbar rendered correctly', () => {
+    const store = mockStore({
+      missions: {
+        missions: [
+          {
+            name: 'Test Mission',
+            description: 'Test Description',
+            reserved: false,
+            id: 'test-id',
+          },
+        ],
+        pending: false,
+        error: false,
+      },
+    });
+    const navbar = render(
+      <Provider store={store}>
+        <Router>
+          <Navbar />
+        </Router>
+      </Provider>,
+    );
+    expect(navbar).toMatchSnapshot();
```

Comment:
- Hey, your snapshot test for the navbar is failing. Please, kindly double-check to fix this issue.

`FAIL  src/__tests__/Navbar.test.js`

Here's a summary of your failing test. please, fix them.

```
Snapshot Summary
 › 1 snapshot failed from 1 test suite. Inspect your code changes or press `u` to update them.
 › 1 snapshot obsolete from 1 test suite. To remove it, press `u`.
   ↳ src/__tests__/Profile.test.js
       • MissionCard MissionCard rendered correctly 1

Test Suites: 2 failed, 5 passed, 7 total
Tests:       3 failed, 9 passed, 12 total
Snapshots:   1 failed, 1 obsolete, 5 passed, 6 total
```